doc/source/ray-core/placement-group.rst

```diff
@@ -146,28 +151,20 @@ Strategy types
 
 Ray currently supports the following placement group strategies:
 
-**STRICT_PACK**
-
-All bundles must be placed into a single node on the cluster.
+**PACK**: All provided bundles are packed onto a single node on a best-effort basis.
 
-**PACK**
+**STRICT_PACK**: All bundles must be placed into a single node on the cluster.
+If strict packing is not feasible, bundles can be placed on multiple `PACK` nodes.
 
-All provided bundles are packed onto a single node on a best-effort basis.
-If strict packing is not feasible (i.e., some bundles do not fit on the node), bundles can be placed onto other nodes nodes.
+**SPREAD**: Each bundle will be spread onto separate nodes on a best effort basis.
```

Comment:
oops. If best effort basis will remain here, please add back in the hyphen (best-effort basis).